lib/atscppapi/examples/globalhook/GlobalHookPlugin.cc

```diff
@@ -19,24 +19,30 @@
 #include <iostream>
 #include <atscppapi/GlobalPlugin.h>
 #include <atscppapi/PluginInit.h>
-
+#include<../ts/Diags.h>
 using namespace atscppapi;
 
 class GlobalHookPlugin : public GlobalPlugin
 {
 public:
   GlobalHookPlugin() { registerHook(HOOK_READ_REQUEST_HEADERS_PRE_REMAP); }
-
+  
```

Comment:
You need to arrange for your editor to clip trailing spaces. This shows up a number of times.